tgstation.dme

```diff
@@ -685,6 +685,7 @@
 #include "code\game\gamemodes\dynamic\dynamic_rulesets_latejoin.dm"
 #include "code\game\gamemodes\dynamic\dynamic_rulesets_midround.dm"
 #include "code\game\gamemodes\dynamic\dynamic_rulesets_roundstart.dm"
+#include "code\game\gamemodes\eldritch_cult\eldritch_cult.dm"

```

Comment:
These files should be renamed since you've rebranded the game mode.

Reply:
done

Reply:
This still reads `eldritch_cult` though?

Reply:
well yeah i didnt rename the files just the datums